testplan/testing/multitest/result.py

```diff
@@ -1404,10 +1404,15 @@ def log(self, message, description=None):
         :type message: ``str`` or instance
         :param description: Text description for the assertion.
         :type description: ``str``
+        :param category: Custom category for the assertion, set to 'email' to
+            include message in email exporter.
+        :type category: ``str``
         :return: ``True``
         :rtype: ``bool``
         """
-        entry = base.Log(message=message, description=description)
+        entry = base.Log(message=message,
```

Comment:
You forgot to run black!

If you are using PyCharm, you can set it up as an external tool to run on every save, it works pretty well: https://black.readthedocs.io/en/stable/editor_integration.html . There are integrations for other editors listed on that page too. Alternatively, you can use the pre-commit hook I added to the internal repo - just copy script/pre-commit to your .git/hooks directory in the root of the OSS repo.